apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -19,6 +19,7 @@
   var requests = {};
   var now = Date.now();
   var ONE_DAY_TIME = 24 * 60 * 60 * 1000;
+  var mediafolder = '/views/shared/test/unit/media/';
```

Comment:
it's just a nit, so do what you want here: I usually prefer to remove the final `/` and put it where you use it -- I think it makes it easier to see it's a directory.

but do what you think.